tests/copilot_helper/test_command_config.py

```diff
@@ -1,13 +1,16 @@
+import pytest
 from click.testing import CliRunner
 
 from dbt_copilot_helper.commands.config import validate
 
 
+@pytest.mark.xdist_group(name="fileaccess")
 def test_running_in_non_copilot_directory():
     result = CliRunner().invoke(validate)
     assert result.output == "Could not find a deployment repository, no checks to run.\n"
 
 
+@pytest.mark.xdist_group(name="fileaccess")
```

Comment:
@lawrence can maybe confirm, but I wouldn't expect to need this flag when we are using `fakefs`, I thought it was unique to each test.

Reply:
@WillGibson Not sure what's going on, but sure looks cool.

Reply:
@acodeninja Am I correct in thinking that `fakefs` is unique to each test case, and therefore the tests can be safely run in parallel?

Reply:
I've removed the decorator from tests using fakefs and the tests are running fine.